package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "lap_three_code_challenge",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-cli serve --mode development --config config/webpack.config.dev.js",
+    "build": "webpack --config config/webpack.config.production.js",
+    "test": "jest --watchAll --silent --setupFilesAfterEnv ./src/test/setupTests.js",
+    "coverage": "jest --silent --setupFilesAfterEnv ./src/test/setupTests.js --coverage --watchAll=false",
+    "autograde": "jest --silent --setupFilesAfterEnv ./src/test/setupTests.js"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
+    },
+    "coveragePathIgnorePatterns": [
+      "/node_modules/",
+      "/actions/",
+      "/reducers/index.js",
+      "/components/index.js"
+    ]
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.13.15",
+    "@babel/plugin-transform-runtime": "^7.12.1",
+    "@babel/preset-env": "^7.13.15",
+    "@babel/preset-react": "^7.13.13",
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^13.1.4",
+    "babel-jest": "^26.6.3",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^5.0.1",
+    "html-webpack-plugin": "^4.5.0",
+    "jest": "^26.6.3",
+    "redux-devtools-extension": "^2.13.9",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.8.0",
+    "webpack-cli": "^4.2.0",
+    "webpack-dev-server": "^3.11.0"
```

Comment:
Webpack and babel are needed to build the website so they should probably be installed as dependencies (since if you run `npm i --only=prod  && npm run build` it breaks

Reply:
i would debate that building the app is done by the developer and those tools are not needed to run the app once it's been built whereas stuff like react is. i decided to look it up found this thread about it: https://stackoverflow.com/questions/40143357

Reply:
Fair enough. I don't really understand it too well myself. I thought I'd test deleting node_modules after building the app and everything runs perfectly fine, and after reading the stackoverflow link the only real  answer I can think of is "it depends" (although the comment about serverless deployments is an interesting one)

Also +1 for the link, didn't realize how many dependency types npm has